yas3fs/__init__.py

```diff
@@ -1211,7 +1211,13 @@ def delete_cache(self, path):
 
     def process_message(self, messages):
         logger.debug("process_message '%s'" % (messages))
-        c = json.loads(messages)
+        # Discard malformed JSON https://github.com/danilop/yas3fs/issues/141
+        try:
+            c = json.loads(messages)
+        except Exception as e:
+            logger.debug("process_message discarding malformed message")
```

Comment:
Leave Exception as-is, on second thought. (`json.loads` only raises `ValueError`, but there's no guarantee that a custom decoder won't raise, say, `AttributeError` or somesuch, and we probably want to be resilient for that case, too.)

Change `logger.debug()` to:
`logger.debug("process_message: malformed message: %s", messages)`